content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+The first thing we'll need to do is add a label to our agent.
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+
+labels:
+  server_type: "webservers"
+
+```
+
+In this example, we've added the label `server_type` with the value `"webservers"`--it's useful to know if a number of agents on our webservers stop reporting in rather than just a single node. Let's add a similar label to a check.
+
+# Check Aggregate
+
+The check we'll use for our aggregate is an http metric check for a ficticious app site 
+
+```yaml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: cpu-check
+  namespace: default
+  labels:
+    app_group: "identity_servers"
+spec:
+  command: "check-http.rb -u https://identity.{{ .entity.name }}.example.com"
+  interval: 10
+  publish: true
+  handlers:
+  - slack
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+
+```
+
+While Sensu is capable of monitoring singular entities, it is also capable of monitoring groups of entities (and checks) through the use of aggregates. _______________ . Let's get started.
+
+===
+
+## Entity Aggregates
+
+In order for a check or entity to be considered as part of an aggregate it must have a label, or set of labels assigned to it. We'll start by setting up a fictional scenario in which we have 20 webservers serving a number of applications. In this scenario, we might not care if a singular server stops responding, but we might care if say, 15 of the 20 stop responding. 
+
+To make an entity part of an aggregate, we would need to add a label in our `/etc/sensu/agent.yml`. For example:
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+name: "webserver01.example.com"
+namespace: "default"
+subscriptions:
+  - webservers
+labels:
+  server_role: "webserver"
+```
+
+After adding the label, we'll make sure to restart our agent to pick up the change in configuration:
+
+`systemctl restart sensu-agent`
+
+We'll need to set up a check that will look at events with the label we've assigned to the entity and ensure that these events are in an OK status. Let's look at an example check:
+
+```yaml
+---
+api_version: core/v2
+type: CheckConfig
+metadata:
+  namespace: default
+  name: webservers-aggregate-check
+spec:
+  runtime_assets:
+  - sensu/sensu-aggregate-check
+  command: sensu-aggregate-check --api-user=foo --api-pass=bar --entity-labels='server_role:webserver' --warn-percent=75 --crit-percent=50
+  subscriptions:
+  - backend
+  publish: true
+  interval: 30
+  handlers:
+  - slack
+  - pagerduty
+  - email
+```
+
+The check command uses a username/password combination to access the API and then matches events with the label "server_role: webserver". In this case, the check will create an event if 75 percent of the aggregate events are in a `warning` state and will also create an event if 50 percent of the aggregate events are in a critical state.
+
+## Check Aggregates
+
```

Comment:
Maybe we could be clearer from the start that this guide explains two separate ways to construct and use aggregates, entity or check (if that is actually what we are explaining -- I may be confused about that).

If we are explaining two options, I also think we should add some basic info about why/when to use one or the other.